contracts/utils/Multicall.sol

```diff
@@ -5,14 +5,17 @@ pragma solidity 0.8.9;
 import { IMulticall } from '../interfaces/IMulticall.sol';
 
 contract Multicall is IMulticall {
+    error MulticallFailed(bytes err);
```

Comment:
I don't like doing this because then the error will not easily be decodable by from the interface.

Reply:
i see, fair enough then